docs/getting-started/full-dotnet.rst

```diff
@@ -0,0 +1,102 @@
+.. include:: /stub-topic.txt
```

Comment:
Consider making the EF6 text red or something. I know it's ugly, but it might make a few more people see it.